perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_virt_elixirBonaFideStatus.java

```diff
@@ -99,7 +134,8 @@ public Attribute getAttributeValue(PerunSessionImpl sess, User user, AttributeDe
 		Attribute attribute;
 
 		if (uesSetAffiliationAttributeMatcher.find() ||	uesRemoveAffiliationAttributeMatcher.find() ||	uesAllAttrsRemovedMatcher.find() ||
-				userSetMatcher.find() || userRemoveMatcher.find() || userAllAttrsRemovedMatcher.find()) {
+				userSetMatcher.find() || userRemoveMatcher.find() || userAllAttrsRemovedMatcher.find() || userPublicationsSetMatcher.find() ||
+				userPublicationsRemoveMatcher.find()) {
```

Comment:
Please change the old names of variables "userSetMatcher" and "userRemoveMatcher". Thank you.